packages/bpk-component-content-cards/README.md

```diff
@@ -0,0 +1,38 @@
+# BpkContentCards
+
+> Backpack example component.
```

Comment:
@olliecurtis will [this URL]( https://backpack.github.io/storybook-prs/2527) still be active after this PR has been merged and deleted? If so I can link to that in the README. As for the docs link, how is the link decided? Is it automated or can we set it? Would be good if we can already decide on what the URL will be so we can link to it already.

Reply:
>@olliecurtis will this URL still be active after this PR has been merged and deleted?

Yes it will be however once it is merged you can use the main deployed version once its been merged of https://backpack.github.io/storybook

>As for the docs link, how is the link decided? Is it automated or can we set it? Would be good if we can already decide on what the URL will be so we can link to it already.

This gets generated when the docs are created so I imagine it would end up being backpack.github.io/components/content-cards

Reply:
missing description of the component here, or just remove it
```suggestion
```